cmake/util.cmake

```diff
@@ -54,3 +57,12 @@ function( debug_try_run msg compile_result compile_output run_result run_output
     message( TRACE "compile_output: '''\n${compile_output}'''" )
     message( TRACE "run_output: '''\n${run_output}'''" )
 endfunction()
+
+#-------------------------------------------------------------------------------
+# assert( condition )
+# Aborts if condition is not true.
+function( assert var )
```

Comment:
I tried to find where this function is used but I failed.
Is it in another commit?

Reply:
It's used in LAPACK++, and `util.cmake` is the same in both repos.
https://github.com/mgates3/lapackpp/blob/cmake/CMakeLists.txt#L554